app/locales/it.json

```diff
@@ -0,0 +1,102 @@
+{
+  "label": {
+    "about_title": "Informazioni",
+    "authorities_add_button_label": "Aggiungi Autorità",
+    "authorities_add_url": "Aggiungi autorità via URL",
+    "authorities_desc": "Seleziona le autorità sanitarie nella tua area per ottenere i dati sulle possibilità di esposizione. Seleziona un nome dal registro globale o inserisci l'indirizzo internet fornito da un'autorità sanitaria che partecipa a Safe Paths",
+    "authorities_input_placeholder": "Incolla l'indirizzo qui",
+    "authorities_no_sources": "Nessuna fonte di dati per ora",
+    "authorities_removal_alert_cancel": "Annulla",
+    "authorities_removal_alert_desc": "Sei sicuro di voler rimuovere questa fonte di dati sanitari?",
+    "authorities_removal_alert_proceed": "Procedi",
+    "authorities_removal_alert_title": "Rimuovi autorità sanitaria",
+    "authorities_title": "Autorità sanitarie",
+    "choose_provider_subtitle": "Per essere informato sulla possibilità di esposizione devi aggiungere un'autorità sanitaria",
+    "choose_provider_title": "Seleziona le autorità sanitarie",
+    "commitment": "Missione",
+    "commitment_para": "Safe Paths registra dati in piena sicurezza e controlla l'intereazione con altre persone usando la tua posizione. I tuoi dati non lasceranno MAI il tuo telefonosenza il tuo consenso",
+    "default_news_site_name": "Novità su Safe Paths",
+    "default_news_site_url": "https://covidsafepaths.org/in-app-news",
```

Comment:
`default_news_site_url` can be removed to fall back to the english file's default.

Reply:
done